js/map.js

```diff
@@ -0,0 +1,58 @@
+import {
+  togglePageActiveState
+} from './form.js';
+import {
+  generateAd
+} from './render.js';
+import {
+  PinSetting,
+  TOKYO_CENTER
+} from './data.js';
+const resetButton = document.querySelector('.ad-form__reset');
+const inputAddress = document.querySelector('#address');
+
+togglePageActiveState(true);
+
+const map = L.map('map-canvas')
+  .on('load', () => {
+    togglePageActiveState(false);
+  })
+  .setView(TOKYO_CENTER, 12);
+
+L.tileLayer(
+  'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+  },
+).addTo(map);
+
+const mainPinIcon = L.icon(PinSetting.main);
+const regularPinIcon = L.icon(PinSetting.regular);
```

Comment:
У нас перечисление констант жеж:
```suggestion
const mainPinIcon = L.icon(PinSetting.MAIN);
const regularPinIcon = L.icon(PinSetting.REGULAR);
```